.gitignore

```diff
@@ -1,2 +1,3 @@
 /_output/
-/mixtool
\ No newline at end of file
+/mixtool
+/.idea
```

Comment:
Rather than adding this to each project, I highly recommend creating a global gitignore for you that always ignores `.idea` folders. 
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Good call, change reverted :)